apps/website/src/components/utils/Footer.astro

```diff
@@ -3,7 +3,9 @@ const props = Astro.props;
 const currentYear = new Date().getFullYear();
 ---
 
-<footer {...props}>© {currentYear} Bentley Systems, Inc</footer>
+<footer class='move-on-visible' data-move-on-visible-direction='down' {...props}>
```

Comment:
do we really need to animate the footer?

Reply:
I feel not animating it might feel abrupt. Because if the user sees that most sections or items animate on becoming visible, they would expect the footer to do the same. If the footer was tall (more than just one sentence) and had a background color, then I feel animating it may not be needed.